tests/core/test_check_manifest.py

```diff
@@ -5,6 +5,11 @@
 
 import pytest
 
+from tests.core.tools import foo
+from tests.core.tools import bar
+from tests.core.tools import baz
+from tests.core.tools.echo import echo
```

Comment:
Can probably just move echo out of its containing directory to simplify path. (also it's nice that we can finally remove this from our actual tools directory!)